dependencies/build_imagemagick.sh

```
@@ -52,20 +59,25 @@ curl -fsSL --retry 2 --retry-delay 5 --retry-max-time 60 -o ./magick.tar.gz \
   --host="${DEB_HOST_GNU_TYPE}" \
-  ${FEATURES}
+  "${FEATURES}"
 
 # Ensure that features are enabled
 for feature in ${FEATURES}
 do
-  grep -- ${feature}'.*yes$' config.log || (echo "Can't enable feature ${feature}"; false)
+  # Be tolerant to minor formatting changes; require the delegate shows final 'yes'
++  # in the "Delegate library configuration" table inside config.log
++  name="${feature#--with-}"
++  name_upper=$(echo "${name}" | tr '[:lower:]' '[:upper:]')
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)